### DISCUSSION TITLE
Freeze column nocodb

### DISCUSSION THREAD

**Discussion by nonewja063** [nocodb/nocodb#6939] [Q&A]
In NOCODB 
How condition to select 1st column in picture ?
and i can change column in 1st ?

![nocodb_freeze_column](https://github.com/nocodb/nocodb/assets/109946116/43a831e9-7e08-4acf-b7b6-07fda9cbe188)

Thank you

**Comment by mertmit** [comment]
Hi @nonewja063,
We refer that column as [Display Value](https://docs.nocodb.com/fields/display-value).
And yes you can change your display value to any other field, see our documentation for detailed steps https://docs.nocodb.com/fields/display-value/#set-display-value
